stardist_environment.yml

```diff
@@ -0,0 +1,12 @@
+name: stardist_environment
+channels:
+  - conda-forge
+dependencies:
+  - pip=21.1.1
+  - python=3.8
+  - pip:
+    - squidpy==1.0.1.dev17+g6ba1291
```

Comment:
@giovp should we specify the version here? Maybe just Squidpy is better?

Reply:
Yeah agree, we can remove the pin, also for python and pip (were probably set by `conda env export`)